frontend/src/v5/store/tickets/tickets.helpers.ts

```diff
@@ -160,17 +160,19 @@ export const isResourceId = (str) => {
 	return regexExp.test(str);
 };
 
-export const getImgSrc = (imgData) => {
-	const { teamspace, project } = useParams<DashboardParams>();
+export const getImgSrcMapFunction = () => {
```

Comment:
This is due to the presence of hooks in the function, so it can get called in the main body of a component, but not inside a method of the latter.

In `TicketImagesList` I need to map the images to their src like this `const imgsSrcs = (value || []).map(getImgSrc);`. This change allows me to do so